src/module_2/analyzing_parquet_files.md

```diff
@@ -0,0 +1,576 @@
+      <td>0.442748</td>
+    </tr>
+    <tr>
+      <th>47</th>
+      <td>medicines-treatments</td>
+      <td>97</td>
+      <td>72</td>
+      <td>0.426036</td>
+    </tr>
+    <tr>
+      <th>45</th>
+      <td>superfoods-supplements</td>
+      <td>159</td>
+      <td>107</td>
+      <td>0.402256</td>
+    </tr>
+    <tr>
+      <th>50</th>
+      <td>low-no-alcohol</td>
+      <td>58</td>
+      <td>39</td>
+      <td>0.402062</td>
+    </tr>
+    <tr>
+      <th>36</th>
+      <td>spirits-liqueurs</td>
+      <td>578</td>
+      <td>350</td>
+      <td>0.377155</td>
+    </tr>
+    <tr>
+      <th>43</th>
+      <td>cider</td>
+      <td>215</td>
+      <td>114</td>
+      <td>0.346505</td>
+    </tr>
+    <tr>
+      <th>24</th>
+      <td>washing-capsules</td>
+      <td>1127</td>
+      <td>581</td>
+      <td>0.340164</td>
+    </tr>
+    <tr>
+      <th>52</th>
+      <td>maternity</td>
+      <td>40</td>
+      <td>20</td>
+      <td>0.333333</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+
+```python
+df_q2.sort_values('abandonment_rate', ascending=False).tail(10)
+```
+
+
+
+
+<div>
+<style scoped>
+    .dataframe tbody tr th:only-of-type {
+        vertical-align: middle;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+
+    .dataframe thead th {
+        text-align: right;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>product_type</th>
+      <th>sold</th>
+      <th>abandoned</th>
+      <th>abandonment_rate</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>21</th>
+      <td>tea</td>
+      <td>1232</td>
+      <td>315</td>
+      <td>0.203620</td>
+    </tr>
+    <tr>
+      <th>40</th>
+      <td>deodorant</td>
+      <td>299</td>
+      <td>75</td>
+      <td>0.200535</td>
+    </tr>
+    <tr>
+      <th>33</th>
+      <td>coffee</td>
+      <td>729</td>
+      <td>175</td>
+      <td>0.193584</td>
+    </tr>
+    <tr>
+      <th>37</th>
+      <td>baby-toddler-food</td>
+      <td>484</td>
+      <td>116</td>
+      <td>0.193333</td>
+    </tr>
+    <tr>
+      <th>8</th>
+      <td>cereal</td>
+      <td>3014</td>
+      <td>700</td>
+      <td>0.188476</td>
+    </tr>
+    <tr>
+      <th>42</th>
+      <td>pet-care</td>
+      <td>238</td>
+      <td>52</td>
+      <td>0.179310</td>
+    </tr>
+    <tr>
+      <th>25</th>
+      <td>bin-bags</td>
+      <td>1083</td>
+      <td>232</td>
+      <td>0.176426</td>
+    </tr>
+    <tr>
+      <th>53</th>
+      <td>adult-incontinence</td>
+      <td>39</td>
+      <td>7</td>
+      <td>0.152174</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>long-life-milk-substitutes</td>
+      <td>6637</td>
+      <td>1134</td>
+      <td>0.145927</td>
+    </tr>
+    <tr>
+      <th>32</th>
+      <td>baby-milk-formula</td>
+      <td>787</td>
+      <td>102</td>
+      <td>0.114736</td>
+    </tr>
+  </tbody>
+</table>
+</div>
+
+
+
+### Insights:
```

Comment:
Nice analysis and insights! Just to add something, I think it would have been better to write the insights in the same section as it corresponding hypothesis. Think of a notebook as a book that someone is going to read: it is easier to follow if you write the conclusion near the question and the data.